drivers/volume/volume.go

```diff
@@ -156,6 +156,22 @@ type Driver interface {
 // StorageProvisionerType provisioner to be used for torpedo volumes
 type StorageProvisionerType string
 
+const (
+	//PortworxStorage portworx storage name
+	PortworxStorage StorageProvisionerType = "portworx"
+	//CsiStorage csi storage name
+	CsiStorage StorageProvisionerType = "csi"
+	//GkeStorage  gce storage name
+	GkeStorage StorageProvisionerType = "gke"
+)
+
+// Provisioners types of supported provisioners
+var Provisioners = map[StorageProvisionerType]StorageProvisionerType{
+	PortworxStorage: "kubernetes.io/portworx-volume",
```

Comment:
These provisioners should be in the driver implementation

Reply:
moved.